isis/src/base/objs/LidarData/LidarData.cpp

```diff
@@ -50,14 +49,32 @@ namespace Isis {
 
 
   /**
-   * Gets the list of Lidar data points.
+   * Gets the list of Lidar data points optionally sorted . 
    *
+   * @sort An option to sort the list.  The default is false (no sort).
```

Comment:
should the sort argument have @param in front of it? That's what I see in other method headers.